contracts/ERC20I.sol

```diff
@@ -13,18 +13,18 @@ pragma solidity ^0.5.0;
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-// 
+//
 // ----------------------------------------------------------------------------
-// Based on the 'final' EIP20 token standard as specified at:
+// Based on the 'final' ERC20 token standard as specified at:
```

Comment:
I think for this we can still keep it as EIP20

Reply:
It really doesnt matter so much, but it's an ERC20 because it is an application level EIP, so in the docs it should just be referred to the same.